### PR TITLE
Add install_qc target

### PR DESCRIPTION
**Pull request by steven-johnson**

### CODE REVIEW

**Comment by pranavb-ca** [comment]
@steven-johnson  - I have built on your changes. I was oversimplifying yesterday that may have given you the impression that we need to copy only two dirs. However, while I was working on adding to your PR, I realized there are some "not-so-holy" things I need to do, such as creating a link to $(PREFIX)/share/halide/tools/GenGen.cpp from $(PREFIX)/tools/GenGen.cpp. So, If you think this is more change than you are willing to accept/maintain, I'll move this to a post-install scripts that we'll maintain internally.

File: Makefile [hunk 1]
```diff
@@ -1631,6 +1639,17 @@ ifeq ($(UNAME), Darwin)
...
 
+# This is a specialized 'install' for users who need Hexagon support libraries as well.
+install_qc: install $(HEXAGON_RUNTIME_LIBS)
+	mkdir -p $(PREFIX)/lib/arm-32-android $(PREFIX)/lib/arm-64-android $(PREFIX)/lib/host $(PREFIX)/lib/v60 $(PREFIX)/tools
+	cp $(ROOT_DIR)/tools/sim_qurt/libsim_qurt.a $(PREFIX)/lib
+	cp $(HEXAGON_RUNTIME_LIBS_DIR)/arm-32-android/* $(PREFIX)/lib/arm-32-android
+	cp $(HEXAGON_RUNTIME_LIBS_DIR)/arm-64-android/* $(PREFIX)/lib/arm-64-android
+	cp $(HEXAGON_RUNTIME_LIBS_DIR)/host/* $(PREFIX)/lib/host
+	cp -r $(HEXAGON_RUNTIME_LIBS_DIR)/v60/* $(PREFIX)/lib/v60
+	ln -sf $(PREFIX)/share/halide/tools/GenGen.cpp $(PREFIX)/tools/GenGen.cpp
```
Comment (steven-johnson) [inline Makefile]: It's at least worth commenting about why the symlinks are needed.

**Comment by steven-johnson** [comment]
I don't see any red flags here, but I ~never use the install target (in local work I ~always rely on specific 'distrib' builds, and inside Google we always build from source).

It would be nice to get feedback from people who use it to see if this fits the model well.

**Comment by steven-johnson** [comment]
LGTM

**Comment by steven-johnson** [comment]
Tagging @abadams to see if he has any concerns before landing

**Comment by abadams** [comment]
lgtm

**Comment by pranavb-ca** [comment]
Thanks for merging, @steven-johnson

**Pull request merged**